cmd/microcloud/main_init.go

```diff
@@ -764,6 +773,17 @@ func (c *initConfig) setupCluster(s *service.Handler) error {
 			}
 		}
 
+		if s.Type() == types.MicroOVN {
+			microOvnBootstrapConf := make(map[string]string)
+			if bootstrapSystem.OVNGeneveAddr != "" {
+				microOvnBootstrapConf["ovn-encap-ip"] = bootstrapSystem.OVNGeneveAddr
+			}
+
+			if len(microOvnBootstrapConf) > 0 {
+				s.SetConfig(microOvnBootstrapConf)
```

Comment:
So we only apply the underlay config during bootstrap/join. When running `microcloud add` or `microcloud service add`, other nodes may not have underlay networking configured, so even though we present a list of every old and new cluster member, no config will change.

I guess this means we also need a way to set the geneve address over the API. 

We could get around this for now if we at least have a way to _get_ the current underlay configuration. We could just skip configuration in the `add` cases if it was not previously configured during `init`.